packages/venia-concept/src/components/ProductOptions/swatch.js

```diff
@@ -1,20 +1,12 @@
 import React, { Component } from 'react';
-import { bool, number, oneOfType, shape, string } from 'prop-types';
+import { bool, func, number, oneOfType, shape, string } from 'prop-types';
 
 import classify from 'src/classify';
 import Icon from 'src/components/Icon';
 import CheckIcon from 'react-feather/dist/icons/check';
 import defaultClasses from './swatch.css';
 
-// TODO: replace with actual swatch colors or images from API
-// M2 GraphQL doesn't currently support them
-const cache = new Map();
-const memoize = fn => key =>
-    cache.has(key) ? cache.get(key) : cache.set(key, fn(key)).get(key);
-
-const getRandomColor = () =>
-    Array.from({ length: 3 }, () => Math.floor(Math.random() * 255)).join(',');
-const memoizedGetRandomColor = memoize(getRandomColor);
+import { memoizedGetRandomColor } from '../../util/getRandomColor';
```

Comment:
Can import from `src/util/getRandomColor` if you don't want the relative paths.

Reply:
Thanks!! I will make this change.